src/Umbraco.Core/Models/PublishedContent/PublishedPropertyType.cs

```diff
@@ -191,11 +192,12 @@ private void InitializeLocked()
                 }
             }
 
+            IDeliveryApiPropertyValueConverter? deliveryApiPropertyValueConverter = _converter as IDeliveryApiPropertyValueConverter;
```

Comment:
I believe our `.editorconfig` states that this variable should be declared as `var` instead of an explicit type.

Reply:
It does indeed, not sure how I missed it.
Solved